akka-docs/src/main/paradox/java/typed.md

```diff
@@ -9,16 +9,58 @@ This module is currently marked as @ref:[may change](common/may-change.md) in th
 
 @@@
 
-As discussed in `actor-systems` (and following chapters) Actors are about
+## Dependency
+
+Akka Typed APIs are bundled in the `akka-typed` artifact.
+Make sure that you have the following dependency in your project:
+
+sbt
+:   @@@vars
+    ```
+    "com.typesafe.akka" %% "akka-typed" % "$akka.version$"
+    ```
+    @@@
+
+gradle
+:   @@@vars
+    ```
+    dependencies {
+      compile group: 'com.typesafe.akka', name: 'akka-typed_2.11', version: '$akka.version$'
+    }
+    ```
+    @@@
+
+maven
+:   @@@vars
+    ```
+    <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-typed_$scala.binary_version$</artifactId>
+      <version>$akka.version$</version>
+    </dependency>
+    ```
+    @@@
+
+## Introduction
+
+As discussed in @ref:[Actor Systems](general/actor-systems.md) (and following chapters) Actors are about
 sending messages between independent units of computation, but how does that
 look like? In all of the following these imports are assumed:
 
-@@snip [IntroSpec.scala]($akka$/akka-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java) { #imports }
+Scala
```

Comment:
Thanks for splitting this out already - this'll make replacing the Java version with a symlink much easier later